pkg/ds/farm.go

```diff
@@ -564,6 +565,25 @@ func (dsf *Farm) spawnDaemonSet(
 
 	desiresCh := ds.WatchDesires(quitSpawnCh, updatedCh, deletedCh)
 
+	go func() {
+		aggregateHealth := replication.AggregateHealth(ds.PodID(), *dsf.healthChecker)
+		ticks := time.NewTicker(1 * time.Minute)
+
+		defer aggregateHealth.Stop()
```

Comment:
`defer`ring these two operations is gratuitous. Could put it under `case <-quitSpawnCh:`; this is proofing it against the probability (whose value is unknown) that this goroutine gains more exit points.